web/src/app/models/user.model.ts

```diff
@@ -1,15 +1,19 @@
+import { SocialConnections } from './socialConnections';
+
 export interface User {
   displayName: string;
   photoUrl: string;
   uid: string;
   creationAt: string;
   lastSignInAt: string;
+  isPrimary: boolean;
```

Comment:
Is this needed? The user will always be primary because the connected socials are secondary.

Reply:
@eddiejaoude When we start handling #65 , this field would be easy check.

Reply:
I think this adds code with no benefit, as checking a small collection is simple also. If you prefer leave it in for now and we can remove it later on

Reply:
Okay @eddiejaoude for now we will leave it, once I start working on #65 and #66 , will get a clear idea about this.